Documentation/contributor-guide/dependency_management.md

```diff
@@ -76,7 +77,10 @@ For mixed cases, in which some modules directly while others indirectly depend o
 We should try to follow the first way, and temporarily fall back to the second one if we run into any issue on the first way. Eventually we
 should fix the issue and ensure all modules depend on the same version of the dependency.
 
-## About gRPC
+## Known incompatible dependency update
+- [build(deps): bump arduino/setup-protoc from 1.3.0 to 2.0.0](https://github.com/etcd-io/etcd/pull/16016)
+
+### About gRPC
```

Comment:
Suggest to add them as two sub items under `Known incompatible dependency updates`.

## Known incomplete dependency updates
### arduino/setup-protoc
Please refer to [build(deps): bump arduino/setup-protoc from 1.3.0 to 2.0.0](https://github.com/etcd-io/etcd/pull/16016).

### About gRPC
......

Reply:
I thought there might be a list of incompatible dependency updates. Using subtitle for a small item like `arduino/setup-protoc` may cause too many subtitles in the future. I don't think it's a big issue so I just move forward with your suggestion and we can always adjust the runbook later.